src/content.config.ts

```
@@ -0,0 +1,74 @@
+      tags: z.array(z.string()).default(["others"]),
+      ogImage: image().or(z.string()).optional(),
+      description: z.string(),
+      canonicalURL: z.string().optional(),
+      hideEditPost: z.boolean().optional(),
+      timezone: z.string().optional(),
+    }),
+});
+
+const projects = defineCollection({
+  loader: file("src/data/projects.json", {
+    parser: (fileContent) => {console.log("@@@@FileContent",fileContent); return JSON.parse(fileContent);},
```

ESLint

[error] Unexpected console statement.

(no-console)